src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java

```diff
@@ -50,9 +50,17 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
                 return true;
             }
         }
-
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+        
+        
+        if (!player.isOp())
++        {
++        	TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
++        	player.setOp(true);
++        }
++        else
++        {
++        	playerMsg("The player you are trying to OP is already OP!");

```

Comment:
Might I suggest changing this to something like 
playerMsg(player.getName() + " is already OP!");

(I havent put that into an IDE so you will need to, but that just looks cleaner I think)